python/constellation/satellites/LeCroy/README.md

```diff
@@ -0,0 +1,65 @@
+---
+# SPDX-FileCopyrightText: 2025 Laurent Forthomme, DESY and the Constellation authors
+# SPDX-License-Identifier: CC-BY-4.0 OR EUPL-1.2
+title: "LeCroy"
+description: "Satellite controlling a LeCroy oscilloscope using the LeCrunch library"
+---
```

Comment:
There are a few additional helpful keys (sorry, undocumented so far!), e.g. "category" would be great

Reply:
I'll add the `category: "Data Sender"`, I guess? As they are undocumented, not sure what is your naming convention :-)

Reply:
Hm given that all the others are currently uncategorized maybe leave it away - sorry for the confusion!

(Looking at https://constellation.pages.desy.de/satellites/index.html)